eng/tools/RepoTasks/RepoTasks.csproj

```diff
@@ -24,6 +24,7 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net472'">
     <PackageReference Include="Wix" Version="3.11.1" />
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
```

Comment:
Huh, was this an unintended change❔ Nothing should need this here especially considering how long this project has sat unchanged and worked fine.

Reply:
It was seemingly picking up this reference from the analyzer package. Since that's no longer being reference, `DownloadFile` complained about a version conflict and recommended adding this reference. That said, isn't there a task that supports downloading in MSBuild now?

Reply:
Ah, your point about transitive dependencies coming in accidentally makes sense. The analyzer has been around for a while.

Nothing to see here 😺 

----

https://docs.microsoft.com/en-us/visualstudio/msbuild/downloadfile-task?view=vs-2019 mentions
> The DownloadFile task is available in MSBuild 15.8 and above only.

I think this means it's not available in `dotnet msbuild`. I don't see a `UsingTask` element providing this task in either `msbuild` or `dotnet msbuild` but I may be missing something. In any case, our task has a special-case fallback to an authenticated URI; the documented one does not. We might be able to figure out a workaround but it's not an immediate priority.